### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by MarsiBarsi**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,21 @@
 
 It is a pre-alpha version of the library. Please, do not use it until official release.
```
Comment (zy2ba) [inline README.md:5]: можно добавить восклицательных знаков для выразительности!!!1111один

**Comment by IKatsuba** [comment]
:tada: This pull request is included in version core@0.1.0-alpha.5 :tada:

The release is available on [GitHub release](<github_release_url>)

**Comment by IKatsuba** [comment]
:tada: This pull request is included in version webpack-plugin@0.1.0-alpha.6 :tada:

The release is available on [GitHub release](<github_release_url>)

**Comment by IKatsuba** [comment]
:tada: This pull request is included in version angular@0.1.0-alpha.8 :tada:

The release is available on [GitHub release](<github_release_url>)

**Pull request merged**